### ISSUE TITLE
bookkeeper nightly release snapshot job failed

### ISSUE THREAD

**Issue by sijie** [apache/bookkeeper#1455]
**BUG REPORT**

1. Please describe the issue you observed:

- What did you do?

CI runs nightly release snapshot job every night.

- What did you expect to see?

CI should pass.

- What did you see instead?

```
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-checkstyle-plugin:3.0.0:check (default-cli) on project bookkeeper-stats-api: Failed during checkstyle execution: Unable to process suppressions file location: bookkeeper/suppressions.xml: Cannot create file-based resource:invalid block type -> [Help 1]
[ERROR]
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR]
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoExecutionException
[ERROR]
[ERROR] After correcting the problems, you can resume the build with the command
[ERROR]   mvn <goals> -rf :bookkeeper-stats-api
```

**Comment by sijie** [comment]
This seems to be related to changes introduced in #1434

**Comment by sijie** [comment]
This is fixed by #1456
